### DISCUSSION TITLE
paddleocr最新版都不处理结果为空的情况吗

### DISCUSSION THREAD

**Discussion by lanmengyiyu** [PaddlePaddle/PaddleOCR#12859] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):

请尽量不要包含图片在问题中/Please try to not include the image in the issue.
![屏幕截图 2023-09-28 162754](https://github.com/PaddlePaddle/PaddleOCR/assets/14342646/097c5d54-1f6b-4dac-98a1-3b954dd87900)

**Comment by SWHL** [comment]
main分支代码已经解决了该问题
